src/ui/component/rewardSummary/view.jsx

```diff
@@ -19,16 +20,15 @@ class RewardSummary extends React.Component<Props> {
         <p className="card__subtitle">
           {fetching && __('You have...')}
           {!fetching && hasRewards ? (
-            <React.Fragment>
-              {/* @i18nfixme */}
-              {__('You have')}
-              &nbsp;
-              <CreditAmount inheritStyle amount={unclaimedRewardAmount} precision={8} />
-              &nbsp;
-              {__('in unclaimed rewards')}.
-            </React.Fragment>
+            <I18nMessage
+              tokens={{
+                credit_amount: <CreditAmount inheritStyle amount={unclaimedRewardAmount} precision={8} />,
+              }}
+            >
+              You have %credit_amount% in unclaimed rewards.
+            </I18nMessage>
           ) : (
-            __('You have no rewards available, please check')
+            <React.Fragment>{__('You have no rewards available, please check')}</React.Fragment>
```

Comment:
`React.Fragment` is only needed if you are returning two or more "things" that aren't contained in a single wrapper, so it isn't needed here.